libnavui-dropin/build.gradle

```diff
@@ -45,6 +45,7 @@ dependencies {
     api project(":libnavui-tripprogress")
     api project(":libnavui-util")
     api project(":libnavui-voice")
+    api project(":libnavui-lifecycle")
```

Comment:
We need to expose the ability to call `MapboxNavigationApp.setup` from the `Application` so we can register onto `registerActivityLifecycleCallbacks`